src/components/Start Choose.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Trojmiasto from "./Trojmiasto";
+import Warszawa from "./Warszawa";
+
+function Start_Choose() {
```

Comment:
I don't get this component, it isn't even used.
I'd remove it.

Reply:
ok